### PR TITLE
Feature/header tests

### PR DESCRIPTION
**Pull request by H-Bo214**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Testing - building tests
- [ ] Formatting - no new features
- [ ] Styling- no new features
#### There are some merge conflicts, but they should be minor.

* Update dependencies
* Removed duplicate className prop from Header button
* Added conditional to componentDidMount to prevent user console error.
* Added test to Header to check initial render state.
* Added test to confirm user image, welcome message for user, logout button
### Files modified:
- [ ] index.html
- [x] App
- [ ] CardContainer
- [ ] PokeCard
- [ ] PokeForm
- [ ] Login
- [ ] FormContainer
- [ ] index.js
- [x] Other files: Header.test.js, Header.js

### CODE REVIEW

File: stretch-auth/src/Header/Header.test.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (jordy1611) [inline stretch-auth/src/Header/Header.test.js:75]: Good rendering tests! We need to test that a function fires on button click as well.

**Review by jordy1611 — APPROVED** [review]
Some of these might fail now that I edited the main page a bit. I didn't remove anything but different buttons have different links now.

File: stretch-auth/src/Header/Header.test.js (hunk 1 above)
Comment (H-Bo214) [reply stretch-auth/src/Header/Header.test.js:75]: Ah yes, I'll update the test.

**Review by AurumValian — APPROVED** [review]
Looks like a good start!

**Pull request merged**